
main() {
  throw Exception("some exception!");
  
  throw 'Hello, this message was thrown.';
}